lib/killbill_client/api/net_http_adapter.rb

```diff
@@ -158,9 +158,6 @@ def request(method, relative_uri, options = {})
             headers = request.to_hash
             headers['authorization'] &&= ['Basic [FILTERED]']
             KillBillClient.log :debug, headers.keys.map { |k| "#{k}='#{headers[k].join(',')}'" }.join(', ')
-            if request.body && !request.body.empty? && request['Content-Type'].include?('application/json')
-              KillBillClient.log :debug, "requestBody='#{request.body}'"
```

Comment:
I agree that debug shouldn't be displayed in normal conditions, but I'm not sure if changing that is feasible or even possible. Also, it seems that `TRACE` is not an [available option](http://api.rubyonrails.org/v3.1/classes/ActiveSupport/BufferedLogger/Severity.html) in most scenarios. I guess Ruby logging options are not as mature as Java's.

I think that, after the inclusion of better logging in the control plugins, this logging became less relevant. If clients need that logging information, they should implement something outside the gem, where they'd have fine grain control of what gets logged and what doesn't.